tests/unit/amo/components/TestLandingPage.js

```diff
@@ -308,9 +308,12 @@ describe(__filename, () => {
     const fakeParams = {
       visibleAddonType: visibleAddonType(ADDON_TYPE_THEME),
     };
+
+    const fakeConfig = getFakeConfig({ enableStaticThemes: false });
+
     const match = { params: fakeParams };
 
-    const root = render({ match });
+    const root = render({ _config: fakeConfig }, { match });
```

Comment:
This one is a little bit more involved. For some reason the filter type is showing as extension. I am not sure why this isnt' failing as is: / Anyway, we should be using the getAndLoadLandingAddons function as it makes it easy to pass in the type and we can use the getAddonTypeFilter to determine which filter to use:

```diff
it('sets the links in each footer for themes', () => {
-  store.dispatch(
-    landingActions.loadLanding({
-      addonType: ADDON_TYPE_THEME,
-      featured: createAddonsApiResult([
-       { ...fakeAddon, name: 'Featured', slug: 'featured' },
-     ]),
-      highlyRated: createAddonsApiResult([
-       { ...fakeAddon, name: 'High', slug: 'high' },
-      ]),
-      trending: createAddonsApiResult([
-        { ...fakeAddon, name: 'Trending', slug: 'trending' },
-      ]),
-    }),
- );

+  const addonType = getAddonTypeFilter(ADDON_TYPE_THEME);

+  _getAndLoadLandingAddons({ addonType });

  const fakeParams = {
    visibleAddonType: visibleAddonType(ADDON_TYPE_THEME),
  };

  const fakeConfig = getFakeConfig({ enableStaticThemes: false });

  const match = { params: fakeParams };

- const root = render({ _config: fakeConfig }, { match });
+ const root = render({ match });

  const addonCards = root.find(LandingAddonsCard);
  expect(addonCards.at(0)).toHaveProp('footerLink', {
    pathname: '/search/',
-    query: { addonType: ADDON_TYPE_THEME, featured: true },
+    query: { addonType, featured: true },
  });
  expect(addonCards.at(1)).toHaveProp('footerLink', {
    pathname: '/search/',
-   query: { addonType: ADDON_TYPE_THEME, sort: SEARCH_SORT_TOP_RATED },
+   query: { addonType, sort: SEARCH_SORT_TOP_RATED },
  });
  expect(addonCards.at(2)).toHaveProp('footerLink', {
    pathname: '/search/',
-   query: { addonType: ADDON_TYPE_THEME, sort: SEARCH_SORT_TRENDING },
+   query: { addonType, sort: SEARCH_SORT_TRENDING },
  });
});

```

also make sure to import the getAddonTypeFilter if it's not already here